bootstrap/bootstrap.go

```diff
@@ -251,3 +255,16 @@ func registerMetrics(metricsManager interfaces.MetricsManager, metrics map[strin
 		lc.Infof("%s metric registered and will be reported (if enabled)", metricName)
 	}
 }
+
+func adaptLogrusBasedLogging(dic *di.Container) {
+	l := container.LoggingClientFrom(dic.Get)
+	// Create a new logger instance
+	hook := &utils.LogrusAdaptor{
+		Logger: l,
+	}
+	logrus.AddHook(hook)
+	logrus.SetFormatter(&logrus.TextFormatter{
+		DisableColors: true,
+	})
+	logrus.SetOutput(io.Discard)
+}
```

Comment:
I liked it better when this was in the utils/logging.go with the others. Only change needed is to pass  LoggingClient instead of dic which get rid of the cyclic import.
```suggestion
func adaptLogrusBasedLogging(lc LoggingClientFrom) {
	// Create a new logger instance
	hook := &utils.LogrusAdaptor{
		Logger: lc,
	}
	logrus.AddHook(hook)
	logrus.SetFormatter(&logrus.TextFormatter{
		DisableColors: true,
	})
	logrus.SetOutput(io.Discard)
}
```

Reply:
i like that idea. much cleaner for sure thx